app/Http/Resources/InvitationLinkResource.php

```diff
@@ -24,7 +24,7 @@ public function toArray($request)
             "id" => $this->id,
             "expiry_date" => $this->expiry_date,
             "permission" => $this->permission,
-            "save_id" => $this->save_id,
+            "save" => new SimplerSaveResource($this->safe),
```

Comment:
isnt it `$this->save` and not `$this->safe` ?

Reply:
nope, it is called safe, because the function to save the model data is already called save()